src/lsp/cobol_ptree/terms.ml

```diff
@@ -97,6 +97,8 @@ let pp_alphanum_string ppf { hexadecimal; quotation; str; runtime_repr } =
   | Simple_quote -> Fmt.pf ppf "'%s'" str
   | Double_quote -> Fmt.pf ppf "\"%s\"" str
 
+let pp_alphanum_string' ppf { payload; _ } = pp_alphanum_string ppf payload
```

Comment:
I think you should use `(pp_with_loc pp_alphanum_string)` instead of that new `pp_alphanum_string'`.

Reply:
I remark that there're a lot of `pp'` functions. Should we only use `pp_with_loc` instead? There're a few inconsistencies and I wasn't sure it was on purpose or something to be changed.
If those functions are unwanted, it may be wiser to delete them to prevent other contributors to use it by copying already existing code as I did.

Reply:
The problem is not necessarily the definition of `pp_alphanum_string'`, it is that it does not use `pp_with_loc`.
Since it was used just once I suggested to avoid the redefinition, but the main issue was that `alphanum_string` was used in the first place.
I've had further thoughts about that, so here you can commit whatever works and I'll adjust later.